packages/neuron-ui/src/components/Settings/general.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
```

Comment:
`general.tsx` file name should be capitalized. So is `wallets`.

Reply:
OK. It is my fault.